Description: patch to work with libc 1.8.0 and gcc 4.7
Bug: http://andybrown.me.uk/wk/2012/04/28/avr-gcc-4-7-0-and-avr-libc-1-8-0-compiled-for-windows/
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=677294
Author: Andy Brown <http://andybrown.me.uk/wk/>
Last-Updated: 2017-05-27
X-Opinion: probably outdated
X-Advice: remove in december 2017

Index: arduino/hardware/arduino/avr/cores/arduino/HardwareSerial.cpp
===================================================================
--- arduino.orig/hardware/arduino/avr/cores/arduino/HardwareSerial.cpp
+++ arduino/hardware/arduino/avr/cores/arduino/HardwareSerial.cpp
@@ -1,3 +1,5 @@
+#define __AVR_LIBC_DEPRECATED_ENABLE__
+
 /*
   HardwareSerial.cpp - Hardware serial library for Wiring
   Copyright (c) 2006 Nicholas Zambetti.  All right reserved.
